picard/ui/ui_options_general.py

```diff
@@ -80,6 +80,9 @@ def setupUi(self, GeneralOptionsPage):
         self.ignore_file_mbids = QtGui.QCheckBox(self.groupBox_2)
         self.ignore_file_mbids.setObjectName(_fromUtf8("ignore_file_mbids"))
         self.verticalLayout.addWidget(self.ignore_file_mbids)
+        self.recursively_add_files = QtGui.QCheckBox(self.groupBox_2)
+        self.recursively_add_files.setObjectName(_fromUtf8("recursively_add_files"))
```

Comment:
Is the general options page really the correct one for this setting? I'm not sure why anyone would want such a feature in the first place, but I imagine it would be better to have it on the Advanced page next to the ignore hidden files checkbox.